ci/Makefile

```diff
@@ -67,20 +67,20 @@ post_run: gather_logs cleanup
 # e2e Tests
 .PHONY: test_upgrade_plan_all_fine
 test_upgrade_plan_all_fine:
-	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_plan_all_fine
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} --verbose --suite test_skuba_upgrade.py --test test_upgrade_plan_all_fine test
 
 .PHONY: test_upgrade_plan_from_previous
 test_upgrade_plan_from_previous:
-	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_plan_from_previous
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} --verbose --suite test_skuba_upgrade.py --test test_upgrade_plan_from_previous test
 
 .PHONY: test_upgrade_apply_all_fine
 test_upgrade_apply_all_fine:
-	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_all_fine
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_all_fine test
 
 .PHONY: test_upgrade_apply_from_previous
 test_upgrade_apply_from_previous:
-	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_from_previous
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_from_previous test
 
 .PHONY: test_upgrade_apply_user_lock
 test_upgrade_apply_all_fine:
-	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_user_lock
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_user_lock test
```

Comment:
I think the `test` subcommand should be before the `--verbose` flag otherwise it breaks I think